python_modules/dagster/dagster/_core/execution/context/invocation.py

```diff
@@ -241,6 +239,8 @@ def bind(
 
         _validate_resource_requirements(self._resource_defs, op_def)
 
+        from dagster._core.definitions.resource_invocation import _resolve_bound_config
```

Comment:
probably make this a non underscore method, I guess lint isn't yelling at you bc this is an in-line import.